spec/pragma.dd

```diff
@@ -149,15 +149,9 @@ pragma(inline)
         )
         $(LI
 ---
-pragma(inline, false)
+pragma(inline, boolean_expression)
```

Comment:
From a grammar point of view, this should be assign expression (e.g. see the [`deprecated` attribute](https://dlang.org/spec/grammar.html#DeprecatedAttribute)):

```suggestion
pragma(inline, boolean_expression)
$(GRAMMAR
$(GNAME DeprecatedAttribute):
    $(D pragma $(LPAREN)) inline $(COMMA) $(GLINK2 expression, AssignExpression) $(D $(RPAREN))
)
```